TicTacToe.cpp

```diff
@@ -49,3 +49,45 @@ bool TicTacToe::remove(int x, int y)
         return true;
     } else return false;
 }
+
+int TicTacToe::checkVictoryHorizontal(int teamSide)
+{
+    for(int y = 0; y <= 2; y = y + 1){
+      for(int x = 0; x <= 2; x = x + 1){
+        if(field[y][x].empty()){
+          field[y][x].push_back(0);
```

Comment:
Could you explain why you're doing this? I'm not sure we want this function to modify the field.